### ISSUE TITLE
[BUG REPORT]: HRRR min/mean values are suspect

### ISSUE THREAD

**Issue by adamscarberry** [USACE/cumulus#291]
Min: -10
Mean: -7.6
Max: 59.6

![image](https://user-images.githubusercontent.com/46494851/185208196-2f63ce7b-a21f-4d82-a912-6c3d97bb5075.png)

**Comment by adamscarberry** [comment]
`gdalinfo -stats hrrr.20220817.t16z.wrfsfcf00.tif`

partial results:

```
GRIB_VALID_TIME=1660752000
STATISTICS_MAXIMUM=61.875
STATISTICS_MEAN=-7.8223246993268
STATISTICS_MINIMUM=-10
STATISTICS_STDDEV=7.532174206883
STATISTICS_VALID_PERCENT=100
```

**Comment by brettpalmberg** [comment]
@adamscarberry, it looks like the magnitude of the statistics from the geotiff are effectively the same as those after packaging in DSS. Therefore, the issue must be in the geoprocess or file naming (maybe grabbing the wrong band out of the file?). Are you concluding the same thing?

**Comment by adamscarberry** [comment]
@brettpalmberg Yes, DSS stats are very close so they are likely coming from the COG.

**Comment by brettpalmberg** [comment]
@adamscarberry, thank you for debugging/isolating the issue to the geoprocess. @jeffsuperglide and I coordinated on the HRRR data this morning, @jeffsuperglide, I assigned the issue to you to complete from here.

Sharing information here for future searchability and awareness:

- Geoprocessing plugins have been moved to the repository https://github.com/USACE/cumulus-geoproc
- Test coverage is in progress for hrrr-total-precip at https://github.com/USACE/cumulus-geoproc/tree/develop/src/tests/integration/hrrr-total-precip. The tests for forecast hour "f00" need to be updated as part of this fix and additional information we discovered in paired-programming. Test updates will be reflected in the PR that fixes this issue (#291).
